src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -35,7 +35,7 @@ public interface IJsonRpcConfig : IConfig
         [ConfigItem(Description = "Port number for JSON RPC calls. Ensure the firewall is configured when enabling JSON RPC.", DefaultValue = "8545")]
         int Port { get; set; }
```

Comment:
we cannot state that by default the same value is used if we stopped using null - it would suggest that when you change JSON RPC and leave WS port default then they would be same.

Reply:
They would be same, if not specified the standard JSON RPC Port is used.